packages/babel-generator/src/generators/classes.js

```diff
@@ -117,6 +117,21 @@ export function ClassProperty(node: Object) {
   this.semicolon();
 }
 
+export function ClassPrivateProperty(node: Object) {
```

Comment:
This should also print the decorators. Also, do Flow and Ts support type annotations?